en/docs/learn/passowrdless-authentication-using-fido2.md

```diff
@@ -0,0 +1,67 @@
+# Passwordless authentication using FIDO2
+
+WSO2 Identity Server supports passwordless authentication using FIDO2, which is a phishing proof 
+passwordless authentication protocol developed as a joint effort between the FIDO Alliance and 
+the World Wide Web Consortium (W3C). 
+
+
+The three major enablers of the FIDO2 flow are;
+
+1. The FIDO2 Authenticator (biometrics, mobile devices or FIDO2 security keys)
+2. The client or browser that operates as a mediator
+3. The WebAuthn Relying Party (WSO2 Identity Server)
+
+<br>
+    
+??? info "Does your browser support your FIDO devices?"
+    
+    The <https://demo.yubico.com/webauthn-technical/registration> site can be used to check the browser
+    support for FIDO devices.
+    
+!!! tip "Before you Begin"
+    WSO2 Identity Server uses the WebAuthn API to enable FIDO-based passwordless authentication. 
+    The WebAuthn API is supported from the following browser versions onwards:
+    
+    -   Chrome(CHROME 67) 
+    -   Firefox (FIREFOX 60)
+    -   Edge (EDGE 17723)
+
+    To define the set of origin URLs where the WSO2 Identity Server User Portal will be hosted (e.g., `https://localhost:9443`):
```

Comment:
Follow the steps given below to define the set of origin URLs where the WSO2 Identity Server User Portal will be hosted